doc/cascadia/profiles.schema.json

```diff
@@ -437,7 +438,7 @@
         },
         "closeOnExit": {
           "default": "graceful",
-          "description": "Sets how the profile reacts to termination or failure to launch. Possible values: \"graceful\" (close when exit is typed or the process exits normally), \"always\" (always close) and \"never\" (never close). true and false are accepted as synonyms for \"graceful\" and \"never\" respectively.",
+          "description": "Sets how the profile reacts to termination or failure to launch. Possible values:\n -\"graceful\" (close when exit is typed or the process exits normally)\n -\"always\" (always close)\n -\"never\" (never close). true and false are accepted as synonyms for \"graceful\" and \"never\" respectively.",
```

Comment:
do newlines look right in VSC?

Reply:
![image](https://user-images.githubusercontent.com/11050425/79016505-09b33500-7b24-11ea-8a08-c03034346149.png)

Reply:
Updated the true/false note to a new line too:
![image](https://user-images.githubusercontent.com/11050425/79021079-767ffc80-7b2f-11ea-9a8f-195c99644616.png)